source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -3134,63 +3110,94 @@ void Graph::propertyEditor()
             }
         }
 
+        const float TEXT_BASE_HEIGHT = ImGui::GetTextLineHeightWithSpacing() * 1.3f;
+        const int SCROLL_LINE_COUNT= 20;
```

Comment:
This can be tweaked as needed. About 20 items seems reasonable now for a scroll area, and the table has no borders. I assume when UI grouping support is added these can be collapsible areas and scroll bars may not be needed as much since each group has less items. For another PR.